packages/x-date-pickers/src/DateCalendar/DateCalendar.tsx

```diff
@@ -5,30 +5,30 @@ import { styled, Theme, useThemeProps } from '@mui/material/styles';
 import { SxProps } from '@mui/system';
 import { unstable_composeClasses as composeClasses } from '@mui/material';
 import { useControlled, unstable_useId as useId, useEventCallback } from '@mui/material/utils';
-import { MonthPicker, MonthPickerProps } from '../MonthPicker/MonthPicker';
 import { useCalendarState } from './useCalendarState';
 import { useDefaultDates, useUtils } from '../internals/hooks/useUtils';
 import { PickersFadeTransitionGroup } from './PickersFadeTransitionGroup';
 import {
-  DayPicker,
-  DayPickerProps,
-  DayPickerSlotsComponent,
-  DayPickerSlotsComponentsProps,
-  ExportedDayPickerProps,
-} from './DayPicker';
+  DayCalendar,
+  DayCalendarProps,
+  DayCalendarSlotsComponent,
+  DayCalendarSlotsComponentsProps,
+  ExportedDayCalendarProps,
+} from './DayCalendar';
+import { MonthCalendar, MonthCalendarProps } from '../MonthCalendar';
+import { YearCalendar, YearCalendarProps } from '../YearCalendar/YearCalendar';
```

Comment:
We should probably unify exporting to either re-export `index` or target the component itself.
So, either
```suggestion
import { MonthCalendar, MonthCalendarProps } from '../MonthCalendar';
import { YearCalendar, YearCalendarProps } from '../YearCalendar';
```
or 
```suggestion
import { MonthCalendar, MonthCalendarProps } from '../MonthCalendar/MonthCalendar';
import { YearCalendar, YearCalendarProps } from '../YearCalendar/YearCalendar';
```